### ISSUE TITLE
ts compilation errors

### ISSUE THREAD

**Issue by cat-ninja** [dlenroc/node-testrail-api#62]
Hi!
I'm facing some errors trying to compile my project:
```
tsc --noEmit
node_modules/@dlenroc/testrail/src/TestRail.ts:589:18 - error TS2304: Cannot find name 'FormData'.

589       body = new FormData();
                     ~~~~~~~~

node_modules/@dlenroc/testrail/src/TestRail.ts:600:30 - error TS2304: Cannot find name 'fetch'.

600       const response = await fetch(url, { method, body, headers });
                                 ~~~~~

node_modules/@dlenroc/testrail/src/TestRail.ts:675:37 - error TS2304: Cannot find name 'FormData'.

675 function appendToFormData(formData: FormData, name: string, value: string | Blob, filename?: string) {
                                        ~~~~~~~~

node_modules/@dlenroc/testrail/src/TestRail.ts:676:45 - error TS2304: Cannot find name 'window'.

676   if (typeof Blob !== 'undefined' && typeof window !== 'undefined' && window?.FormData === FormData) {
                                                ~~~~~~

node_modules/@dlenroc/testrail/src/TestRail.ts:676:71 - error TS2304: Cannot find name 'window'.

676   if (typeof Blob !== 'undefined' && typeof window !== 'undefined' && window?.FormData === FormData) {
                                                                          ~~~~~~

node_modules/@dlenroc/testrail/src/TestRail.ts:676:92 - error TS2304: Cannot find name 'FormData'.

676   if (typeof Blob !== 'undefined' && typeof window !== 'undefined' && window?.FormData === FormData) {
                                                                                               ~~~~~~~~
Found 6 errors in the same file, starting at: node_modules/@dlenroc/testrail/src/TestRail.ts:589
```

It looks like the file is missing imports or something:
```ts
import { Request, Response } from './payload';
import { stringify as qs } from './query-string';
import { TestRailException } from './TestRailException';

export * from './payload';
export type { Request as Payload, Response as Model };

class TestRail {
  static Exception = TestRailException;
  private readonly username?: string;
  private readonly password?: string;
  private readonly baseURL: string;

  constructor(config?: { host: string; username: string; password: string }) {
    // @ts-ignore - Backward compatibility
    this.username = config?.username || config?.user;
    this.password = config?.password;
    this.baseURL = (config?.host || '') + '/index.php?/api/v2/';
  }
```

**Comment by dlenroc** [comment]
Hi @cat-ninja, I recently released version [1.7.2](https://www.npmjs.com/package/@dlenroc/testrail/v/1.7.2), which improved [ts support](https://github.com/dlenroc/node-testrail-api/pull/63), and would like to know if it solved this problem ?

**Comment by cat-ninja** [comment]
> Hi @cat-ninja, I recently released version [1.7.2](https://www.npmjs.com/package/@dlenroc/testrail/v/1.7.2), which improved [ts support](https://github.com/dlenroc/node-testrail-api/pull/63), and would like to know if it solved this problem ?

Hi @dlenroc. This worked like a charm - I don't see any issues. Thanks!
